op-e2e/actions/upgrades/interop_fork_test.go

```diff
@@ -0,0 +1,144 @@
+package upgrades
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/helpers"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/interop/dsl"
+	"github.com/ethereum-optimism/optimism/op-e2e/actions/upgrades/utils"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/derive"
+	"github.com/ethereum-optimism/optimism/op-node/rollup/event"
+	"github.com/ethereum-optimism/optimism/op-service/predeploys"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInteropUpgrade(gt *testing.T) {
+	t := helpers.NewDefaultTesting(gt)
+
+	// Run Isthmus until the interop upgrade
+	is := dsl.SetupInterop(t, dsl.SetInteropOffsetForAllL2s(uint64(15)))
```

Comment:
Nit: be really nice for readability to assert the L2 block time as you drive the L2 chain forward during the test. Then it can be really obvious when you are crossing the activation time.

Reply:
Agreed, will fix in follow-up change.